### PR TITLE
Reorgainze ParseUI

### PR DESCRIPTION
**Pull request by wangmengyan95**

### CODE REVIEW

**Comment by wangmengyan95** [comment]
@grantland, it seems developers are OK with this change. Rebase on latest master. Please take a look,

File: ParseUI-Widget/build.gradle [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *  FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ *  COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ *  IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ *  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+apply plugin: 'com.android.library'
+
+dependencies {
+    compile 'com.android.support:support-v4:22.0.0'
```
Comment (grantland) [inline ParseUI-Widget/build.gradle]: What widgets require this?

File: ParseUI-Widget/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  ~  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+  ~  FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+  ~  COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+  ~  IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+  ~  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+  -->
+
+<manifest package="com.parse">
```
Comment (grantland) [inline ParseUI-Widget/src/main/AndroidManifest.xml]: should this be `com.parse.widget`?

File: README.md [hunk 3]
```diff
@@ -1,87 +1,35 @@
...
-We built several sample apps demonstrating how to use the `ParseLoginUI` library.  Before importing
-this library into your app, we recommend that you run these sample apps to become familiar with its
-functionality and customizations.  In this section, we describe Android Studio instructions for
-running the sample apps (as a standalone project), and importing the `ParseLoginUI` library into
-your own app.  These instructions were last tested on Android Studio 1.1.0.
+The `ParseUI-Login` library is used for building login and signup flows with the Parse Android SDK. You can easily configure the look and feel of the login screens by either specifying XML configurations or constructing an Intent in code.
 
-#### Running Sample Projects
-To run our sample apps, you need to import this repo as a standalone Gradle project:
+The `ParseUI-Widget` library provides several useful UI widgets which integrate with `ParseObject` seamlessly.
```
Comment (grantland) [inline README.md]: > ... integrate with the [Parse SDK](https://github.com/ParsePlatform/Parse-SDK-Android) seamlessly

**Comment by grantland** [comment]
I think you still need to clean `/ParseLoginUI`

**Comment by grantland** [comment]
LGTM, feel free to squash and merge!

**Pull request merged**